### PR TITLE
Updated almost everything - DragonPvP

### PR DESCRIPTION
**Pull request by SonoAX**
## Type of change

- [ ] Add new directory for a server
- [x] Update directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/dragonpvp/background.png
Comment (DoJapHD) [inline minecraft_servers/dragonpvp/background.png:1]: Compress image

File: minecraft_servers/dragonpvp/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/dragonpvp/background@2x.png:1]: Compress image

**Review by DoJapHD — CHANGES_REQUESTED** [review]
Please change web_support to a FAQ, contact form or something similar; web_support can't be a forum

**Comment by SonoAX** [comment]
> Please change web_support to a FAQ, contact form or something similar; web_support can't be a forum

Can it be a redirect to our discord ticket support channel?

File: minecraft_servers/dragonpvp/manifest.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "social": {
       "web": "https://www.dragonpvp.site",
       "web_shop": "https://www.dragonpvp.site/store",
-      "web_support": "https://www.dragonpvp.site/forum",
+      "web_support": "https://www.dragonpvp.site/supporto",
```
Comment (DoJapHD) [inline minecraft_servers/dragonpvp/manifest.json:11]: Also cannot be just text "join Discord"; remove if something like a FAQ is non-existent

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**